src/main/java/tech/gruppone/stalker/server/controllers/PasswordController.java

```diff
@@ -0,0 +1,69 @@
+package tech.gruppone.stalker.server.controllers;
+
+import java.io.IOException;
+import javax.mail.MessagingException;
+import lombok.Value;
+import lombok.extern.log4j.Log4j2;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import reactor.core.publisher.Mono;
+import tech.gruppone.stalker.server.configuration.SmtpMailSenderConfiguration;
+import tech.gruppone.stalker.server.exceptions.EmailErrorException;
+import tech.gruppone.stalker.server.repositories.UserRepository;
+
+@Value
+@RestController
+@Log4j2
+@RequestMapping
+public class PasswordController {
+
+  /*
+    FOLLOW THIS STEPS to work properly with this endpoint and to not get an MailAuthenticationException error:
+    Since we are using Gmail to send email, we need to enable "less secure access" on the Gmail account.
+    - Log out from all the Google accounts from the browser. Login in to the gmail account configured in application.properties.
+    - Then click on https://myaccount.google.com/lesssecureapps and turn on access for less secure apps.
+     - Alternately, if the link up doesn't work, go on https://accounts.google.com/b/0/DisplayUnlockCaptcha.
+    Now, you invoke /user/password/recovery endpoint and you'll receive an mail by GruppOne!
+  */
+
+  private UserRepository userRepository;
+
+  @Autowired private SmtpMailSenderConfiguration smtpMailSender;
+
+  // TODO move this to the corresponding service (not a controller's responsibility)
+  private static final String EMAIL_BODY =
+      String.join(
+          System.lineSeparator(),
+          "You're received this mail because you want to recovery your Stalker's password.",
```

Comment:
"You're receiving this email because you asked to recover your Stalker account's password."